Runtime/API.cs

```diff
@@ -10,14 +10,43 @@ namespace StringSDK
 {
     public static class Constants
     {
-        public const string api_base_url = "http://localhost:5555";
+        public const string PROD_API_URL = "https://api.string-api.xyz";
+        public const string SANDBOX_API_URL = "https://api.sandbox.string-api.xyz";
+        public const string DEV_API_URL = "https://string-api.dev.string-api.xyz";
+        public const string LOCAL_API_URL = "http://localhost:5555";
```

Comment:
for SDK we dont want to add local env, or at least not expose an option for local.